apm-sniffer/apm-sdk-plugin/kafka-plugin/src/main/java/org/apache/skywalking/apm/plugin/kafka/ProducerConstructorMapInterceptor.java

```diff
@@ -22,16 +22,27 @@
 import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.InstanceConstructorInterceptor;
 import org.apache.skywalking.apm.util.StringUtil;
 
+import java.util.List;
 import java.util.Map;
+import java.util.regex.Pattern;
 
+/**
+ * Target at kafka-clients >=2.1.0
```

Comment:
Still confused what your new comments. Isn't 2.0 even 1.x client libraries using this interceptor?

Reply:
> Still confused what your new comments. Isn't 2.0 even 1.x client libraries using this interceptor?

I see. You are right. It can be used even below 2.0, if users intend to use Map type constructors.

I have removed this comment.